lms/templates/navigation-edx.html

```diff
@@ -1,156 +0,0 @@
-## mako
```

Comment:
Reviewers: our analysis showed that this file is no longer used, and we verified this with @mattdrayer and team.